MvvmCross-Plugins/PhoneCall/MvvmCross.Plugins.PhoneCall.Droid/MvxPhoneCallTask.cs

```diff
@@ -22,8 +22,10 @@ public class MvxPhoneCallTask
         public void MakePhoneCall(string name, string number)
         {
             string phoneNumber;
-            if (Build.VERSION.SdkInt >= BuildVersionCodes.Lollipop)
+            if (Build.VERSION.SdkInt >= BuildVersionCodes.N)
                 phoneNumber = PhoneNumberUtils.FormatNumber(number, Locale.GetDefault(Locale.Category.Format).Country);
+            else if (Build.VERSION.SdkInt >= BuildVersionCodes.Lollipop)
+                phoneNumber = PhoneNumberUtils.FormatNumber(number, Locale.Default.Country);
```

Comment:
Both the N and the Lollipop version call the same method signature here, what are you protecting here?